### PR TITLE
Automatic publication

### PR DESCRIPTION
**Pull request by plehegar**
For https://github.com/w3c/WebPlatformWG/issues/88

***
[Preview](https://s3.amazonaws.com/pr-preview/plehegar/webidl/tr-publish.html) | [Diff](https://s3.amazonaws.com/pr-preview/heycam/webidl/d6c3ddb...plehegar:03fa97e.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -9,6 +9,7 @@ Mailing List Archives: https://lists.w3.org/Archives/Public/public-script-coord/
 Repository: heycam/webidl
 !Feedback: <a href="https://github.com/heycam/webidl">GitHub</a> (<a href="https://github.com/heycam/webidl/issues/new">new issue</a>, <a href="https://github.com/heycam/webidl/issues">open issues</a>, <a href="https://www.w3.org/Bugs/Public/buglist.cgi?product=WebAppsWG&amp;component=WebIDL&amp;resolution=---">legacy bug tracker</a>)
 ED: https://heycam.github.io/webidl/
+TR: https://www.w3.org/TR/webidl-2/
```
Comment (tobie) [inline index.bs]: Can we have this point to https://www.w3.org/TR/webidl/ instead?

File: deploy.sh [hunk 2]
```diff
@@ -62,3 +68,9 @@ git commit -m "Deploy to GitHub Pages: ${SHA}"
 
 # Now that we're all set up, we can push.
 git push $SSH_REPO $TARGET_BRANCH
+
+function TRpush {
+  curl "https://labs.w3.org/echidna/api/request" --data "url=https://heycam.github.io/webidl/echidna.html" --data "decision=https://lists.w3.org/Archives/Public/public-webapps/2014JulSep/0627.html" --data "token=$TOKEN"'
```
Comment (tobie) [inline deploy.sh]: where is `$TOKEN` defined?

Comment (tobie) [inline deploy.sh]: That implies keeping https://heycam.github.io/webidl/echidna.html around, right? Isn't there a cleaner option, here? Not a deal breaker, but the fewer things are floating around, the better.

File: .travis.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ env:
   global:
     - ENCRYPTION_LABEL: "de8b5a0f10ef"
     - COMMIT_AUTHOR_EMAIL: "cam@mcc.id.au"
+    - secure: "IagnnmSy+JOF/Dqv25lSabCLeW5Rxh2VV6MJZjZuRhZR+CDTmH+ZdskHmk0TC19DH2bJ4t+LpBYrVjrCBu+6Ow923i3lNTQiVlXNIvTEZyWKcgttiTDjhytby6gE/CyuM1tPEC34IBnTUuWdURBHe7XFMCGQRDUD02troYxv4b1Jvv6+JzYtE90v6WVv0b0LjiNJ0ruftP7btjMypJoN40N5ObqNwIecL5RfJeEZoE01LtdrpxBEPSgPkv3VN+Fscjwpi8IASbeYLy294KDjRO1WOHx6i02DvJPnTLHdS0LUU6vBcLGRcz0+d760LPsExNRyJ4xpehewux0ueSXm2oMnujdWK0v9eUPrsoimzwL69c5yzCCAr+1PMVffOLAH8kcC2lklZ84LTvy3FGchh5SMfoXlgMwWoJSH4bub2yr38TB/skOem6OuO2M2PUdWkzAMRcj24LZhvCNSfkvy86q1ShtUFRG2mKLjmZa6FK0ONVUVeRbEjZ8PALM76g07c+uZcOAGu5A3xKoP2KOe0YR4rS4T++9oW8+svQDF4GiA5KqZiB8Yce6mW9NQ7/LQZpjllQtqiTATryJ3LW2yRzdo7k/Mz+ajHLhhiElKzTreRHTLZ/8YHvcL1wud1bDCAtoTOem/v3mxGHiXD6NvJZF6hz/Gf0guhN1yigHlNFI="
```
Comment (tobie) [inline .travis.yml]: I'm not too sure how this is supposed to work. Is this supposed to turn into `$TOKEN` later on? Or is there some other magic happening I'm not aware of here.

File: deploy.sh (hunk 2 above)
Comment (plehegar) [inline deploy.sh]: this avoids having to setup a hook. the file would only shows in the gh-pages branch.

**Comment by tobie** [comment]
@heycam sounds like you don't need to do anything for this to work.

File: .travis.yml (hunk 3 above)
Comment (plehegar) [inline .travis.yml]: it's indeed the TOKEN variable encrypted with travis encrypt.

File: deploy.sh (hunk 2 above)
Comment (plehegar) [inline deploy.sh]: in .travis.yml (secure).

Comment (tobie) [inline deploy.sh]: So Travis does the magic of converting one to the other, right?

Comment (tobie) [inline deploy.sh]: Oh, if a hook-based solution avoids this, and @heycam is reachable, I'd rather we go that route.

File: index.bs (hunk 1 above)
Comment (plehegar) [inline index.bs]: The short answer is yes. Looking to see if bikeshed can accommodate https://github.com/tabatkins/bikeshed/issues/1064 .

Comment (plehegar) [inline index.bs]: (to be clearer, ideally, I'd like to get rid of that line altogether)

File: deploy.sh (hunk 2 above)
Comment (plehegar) [inline deploy.sh]: correct

Comment (plehegar) [inline deploy.sh]: A hook-based solution would require additional services. this seems overkill for this...

Comment (tobie) [inline deploy.sh]: Oh--misunderstood your previous comment. Sorry.

**Pull request closed without merging**